tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -725,16 +725,54 @@ func (a *AwsProvider) getPublicSubnetId(infraID string, vpc *ec2.Vpc) (string, e
 		return "", err
 	}
 
+	// Get the instance offerings that support Windows instances
+	scope := "Availability Zone"
+	productDescription := "Windows"
+	f := false
+	offerings, err := a.EC2.DescribeReservedInstancesOfferings(&ec2.DescribeReservedInstancesOfferingsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("scope"),
+				Values: []*string{&scope},
+			},
+		},
+		IncludeMarketplace: &f,
```

Comment:
Since we are not using `f` anywhere else, can we just provide this value as false, with reasoning?

Reply:
we can use to.BoolPtr(false) or for string to.StringPtr("example"), but internally it also uses the same approach i.e sending the address of the variable.

Reply:
I am happy to do that